nncf/common/pruning/mask_propagation.py

```diff
@@ -40,7 +39,7 @@ def __init__(self, graph: NNCFGraph, pruning_operator_metatypes: PruningOperatio
         self._graph = graph
         self._pruning_operator_metatypes = pruning_operator_metatypes
 
-    def get_meta_operation_by_type_name(self, type_name: str) -> DefaultMetaOp:
+    def get_meta_operation_by_type_name(self, type_name: str) -> 'DefaultMetaOp':
```

Comment:
Did this to prevent import cycling. Is that ok? @alexsu52 @shamp

Reply:
I'm against this, if this is avoidable. Commonly it is avoidable by adding more .py files and extracting the common dependencies/structs that themselves have little or no dependencies into these separate files. This also helps split the larger files into smaller, more human-friendly ones.

Reply:
Ok, I'll refactor it

Reply:
Done